### DISCUSSION TITLE
new versions release

### DISCUSSION THREAD

**Discussion by Au-angeloupali** [spacedriveapp/spacedrive#2834] [General]
I am a Windows user. I go to the site daily to see if a new version has been released.
I am waiting for the column view (Mac finder style) and acrylic background integration.

Do you have any information regarding a release date for a new version or specifically for these features? I know you are caught up in other projects, but a timeline would be appreciated.

Thank you Angelo.

**Comment by Au-angeloupali** [comment]
@utkubakir please can i have a response?

**Comment by fogodev** [comment]
Hi there, the next release should arrive soon, we're implementing some P2P support on it, alongside some cloud features the already implemented and a bunch of really crucial bugfixes. Column view is planned, but isn't a top priority right now, so it won't land on this one. But should arrive soon afterwards nonetheless.

**Comment by Au-angeloupali** [comment]
@fogodev any news on the release date?

**Comment by Au-angeloupali** [comment]
@fogodev you have good news for me, the column view and the acrylic background on the left side
:) :)

**Comment by Sergey842248** [comment]
is there anything new?
